README.md

```diff
@@ -19,75 +19,117 @@
   <a href="https://github.com/meilisearch/MeiliSearch/discussions" alt="Discussions"><img src="https://img.shields.io/badge/github-discussions-red" /></a>
 </p>
 
-<p align="center">
-  ⚡ Lightning Fast, Ultra Relevant, and Typo-Tolerant Search Engine MeiliSearch client written in Java
-</p>
+<p align="center">⚡ The MeiliSearch API client written for Java</p>
 
-<hr>
+**MeiliSearch Java** is a client for **MeiliSearch** written in Java. **MeiliSearch** is a powerful, fast, open-source, easy to use and deploy search engine. Both searching and indexing are highly customizable. Features such as typo-tolerance, filters, facets and synonyms are provided out-of-the-box.
 
 ### ⚠️ Important!: this is WIP, and not available for production ⚠️
 
 ## Table of Contents <!-- omit in toc -->
 
+- [📖 Documentation](#-documentation)
 - [🔧 Installation](#-installation)
-- [🚀 Getting started](#-getting-started)
+- [🚀 Getting Started](#-getting-started)
 - [🤖 Compatibility with MeiliSearch](#-compatibility-with-meilisearch)
-- [🎬 Examples](#-examples)
+- [💡 Learn More](#-learn-more)
 - [⚙️ Development Workflow and Contributing](#️-development-workflow-and-contributing)
 
-# meilisearch-java
+## 📖 Documentation
 
-Java client for MeiliSearch.
-
-MeiliSearch provides an ultra relevant and instant full-text search. Our solution is open-source and you can check out [our repository here](https://github.com/meilisearch/MeiliSearch).
-
-Here is the [MeiliSearch documentation](https://docs.meilisearch.com/) 📖
+See our [Documentation](https://docs.meilisearch.com/guides/introduction/quick_start_guide.html) or our [API References](https://docs.meilisearch.com/references/).
 
 
 ## 🔧 Installation
 
 // TODO:
 
 
-## 🚀 Getting started
+## 🚀 Getting Started
+
+#### Add documents <!-- omit in toc -->
 
-#### Quickstart
 ```java
+import com.meilisearch.sdk.Client;
 import com.meilisearch.sdk.Config;
+import com.meilisearch.sdk.Index;
+
+public class Main {
+
+	public static void main(String[] args) throws Exception {
+
+		final String documents = "["
+			+ "{\"book_id\": 123, \"title\": \"Pride and Prejudice\"},"
+			+ "{\"book_id\": 456, \"title\": \"Le Petit Prince\"},"
+			+ "{\"book_id\": 1, \"title\": \"Alice In Wonderland\"},"
+			+ "{\"book_id\": 1344, \"title\": \"The Hobbit\"},"
+			+ "{\"book_id\": 4, \"title\": \"Harry Potter and the Half-Blood Prince\"},"
+			+ "{\"book_id\": 2, \"title\": \"The Hitchhiker\'s Guide to the Galaxy\"}"
+			+ "]";
+
+		Client client = new Client(new Config("http://localhost:7700", "masterKey"));
+		Index index = client.createIndex("books");
```

Comment:
Even if it's going to be changed, could you add the line with the `getIndex`? Like in [this README](https://github.com/meilisearch/meilisearch-python#-getting-started)